GitSocial/GitSocial/GSButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  GSButton.swift
+//  GitSocial
+//
+//  Created by Dillon on 11/27/21.
+//
+
+import UIKit
+
+class GSButton: UIButton {
```

Comment:
We should probably rename this to `GSTextButton` to eliminate confusion if we want to create a custom button that only uses an image. Doing so would mean adding a second (or more) subclass for an image only button. Another solution might be to implement another initializer, and different configuration methods based on the type of button desired. Thoughts?
```suggestion
class GSTextButton: UIButton {
```

Reply:
Opened issue #5 for further discussion